lib/options.js

```diff
@@ -108,6 +108,12 @@ module.exports = optionator({
         description: "Report errors only"
     },
     {
+        option: "max-warnings",
+        type: "Number",
+        default: "-1",
+        description: "Show an error if too many warnings"
```

Comment:
"Nonzero" or "error" exit code? I suppose we can assume our users are tech-savvy, but I always like to avoid specific numbers. If you prefer nonzero I will go with that.